src/Framework/CompatibilitySuppressions.xml

```diff
@@ -1,6 +1,181 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- https://learn.microsoft.com/dotnet/fundamentals/package-validation/diagnostic-ids -->
 <Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
```

Comment:
We'll probably have to add all of these back, but I like having the file here to call out the changes for discussion